### ISSUE TITLE
Extend RouteModel, or add new layer, to handle compound routes

### ISSUE THREAD

**Issue by erikquinn** [openscope/openscope#81]
_From @n8rzz on November 21, 2016 2:16_

RouteModel only supports `entry.procedure.exit`.  It will need to be extended, or have a new layer added, to support the following:
```
Input: "KSFO.OFFSH9.SXC.V458.IPL.J2.JCT..LLO..ACT..KACT"
Return: ["KSFO.OFFSH9.SXC", "SXC.V458.IPL", "IPL.J2.JCT", "LLO", "ACT", "KACT"]
```

- [ ] Create `RouteFactory` that can determine if a route is simple or compund
- [ ] `CompundRouteModel` is a collection of `SimpleRouteModel`s 
- [ ] rename `RouteModel` to `SimpleRouteModel`
- [ ] Update implementations of `RouteModel` if required
- [ ] Add new tests

see comments if [n8rzz/atc#48](https://github.com/n8rzz/atc/pull/48) for more info on route structure

_Copied from original issue: n8rzz/atc#130_

---

Blocked by #193.

**Comment by erikquinn** [comment]
Isn't this implemented in #139, albeit in a completely different way than as described here?

**Comment by n8rzz** [comment]
It is, this will be closed once we merge #139
